pkg/artifacts/fetch.go

```diff
@@ -19,27 +19,34 @@ type ArtifactType string
 
 const (
 	TypeJson ArtifactType = "type:json"
+	TypeJsonArray ArtifactType = "type:json-array"
```

Comment:
interesting indentation, will this pass a linter?

Reply:
> will this pass a linter?
sure ! .... there is no linter :#

I come from a world where "value alignment" is discouraged ;-)
> If an assignment has a right hand side, then the equality sign should have exactly one space on both sides
https://www.python.org/dev/peps/pep-0008/#id53

fixed

we'll have to add a liner eventually, 
in the GPU Operator I've seen that they lint on `make`, I'll have a look at that